libraries/core_libs/network/src/tarcap/packets_handlers/votes_sync_packet_handler.cpp

```diff
@@ -61,16 +61,17 @@ void VotesSyncPacketHandler::process(const PacketData &packet_data, const std::s
                      << ") already saved in queue.";
         continue;
       }
+      auto vote_ptr_copy = vote_n;
```

Comment:
same here. remove "vote_ptr_copy", and remove "std::move" in "if (!vote_mgr_->addUnverifiedVote(std::move(vote_n)))"

Reply:
same answer